src/main/java/uk/gov/hmcts/reform/coh/schedule/notifiers/EventNotifierJob.java

```diff
@@ -88,13 +90,36 @@ public void execute() {
                         log.info(String.format("Found task %s to handle %s.", task.getClass(), sessionEventType.getEventTypeName()));
                         task.execute(sessionEvent.getOnlineHearing());
                     }
+                    sessionEventService.updateSessionEvent(sessionEvent);
+                } else {
+                    log.error(String.format("Unable to send notification to endpoint: %s", register.getForwardingEndpoint()));
+                    doFailureUpdate(register, sessionEvent);
                 }
-            } catch (NotificationException e) {
+            } catch (Exception e) {
+                doFailureUpdate(register, sessionEvent);
                 log.error(String.format("Exception while trying to send a notification. Exception is %s", e.getMessage()));
             }
         }
     }
 
```

Comment:
doFailureUpdate will update the counter and set the forwarding state to failed when the counter exceeds (1+retries) attempts